modules/services/wpaperd.nix

```
@@ -3,12 +3,12 @@
 with lib;
 
 let
-  cfg = config.programs.wpaperd;
+  cfg = config.services.wpaperd;
   tomlFormat = pkgs.formats.toml { };
 in {
   meta.maintainers = [ hm.maintainers.Avimitin ];
 
```

Something like this should help point people using the old names in the right direction:

```suggestion

  imports = [
    (mkRenamedOptionModule # \
      [ "programs" "wpaperd" "enable" ] # \
      [ "services" "wpaperd" "enable" ])

    (mkRenamedOptionModule # \
      [ "programs" "wpaperd" "package" ] # \
      [ "services" "wpaperd" "package" ])

    (mkRenamedOptionModule # \
      [ "programs" "wpaperd" "settings" ] # \
      [ "services" "wpaperd" "settings" ])
  ];

```